### PR TITLE
Display borrowable balance on borrow form

### PR DESCRIPTION
**Pull request by therealemjy**
- update `formatCoinsToReadableValue` so it only uses shorthand when `shorthand` param is passed as `true`
- display borrowable token amount on borrow form

### CODE REVIEW

File: src/pages/Dashboard/Modals/BorrowRepay/Borrow/index.spec.tsx [hunk 1]
```diff
@@ -41,6 +41,65 @@ describe('pages/Dashboard/BorrowRepayModal/Borrow', () => {
...
+      </AuthContext.Provider>,
+    );
+
+    const borrowDeltaDollars = fakeUserTotalBorrowLimitDollars.minus(
+      fakeUserTotalBorrowBalanceDollars,
+    );
+    const borrowDeltaTokens = borrowDeltaDollars.dividedBy(fakeAsset.tokenPrice);
+
+    await waitFor(() =>
+      getByText(`${borrowDeltaTokens.toFixed()} ${customFakeAsset.symbol.toUpperCase()}`),
```
Comment (coreyar) [inline src/pages/Dashboard/Modals/BorrowRepay/Borrow/index.spec.tsx]: nit: `symbol` should always be the uppercase displayable value:

File: src/pages/Dashboard/Modals/BorrowRepay/Borrow/index.tsx [hunk 2]
```diff
@@ -102,6 +116,20 @@ export const BorrowForm: React.FC<IBorrowFormProps> = ({
               hasError={errors.amount === ErrorCode.HIGHER_THAN_MAX}
             />
 
+            <Typography
```
Comment (coreyar) [inline src/pages/Dashboard/Modals/BorrowRepay/Borrow/index.tsx]: nit: I have done this in some places but the `TokenTextField` has a description prop for positioned text at the bottom left of the component.

**Review by coreyar — APPROVED** [review]

File: src/pages/Dashboard/Modals/BorrowRepay/Borrow/index.tsx (hunk 2 above)
Comment (therealemjy) [reply src/pages/Dashboard/Modals/BorrowRepay/Borrow/index.tsx]: Ah right, I forgot! Good point.

File: src/pages/Dashboard/Modals/BorrowRepay/Borrow/index.spec.tsx (hunk 1 above)
Comment (therealemjy) [reply src/pages/Dashboard/Modals/BorrowRepay/Borrow/index.spec.tsx]: Ok nice, I wasn't sure

File: src/pages/Dashboard/Modals/BorrowRepay/Borrow/index.tsx (hunk 2 above)
Comment (therealemjy) [reply src/pages/Dashboard/Modals/BorrowRepay/Borrow/index.tsx]: Fixed.

File: src/pages/Dashboard/Modals/BorrowRepay/Borrow/index.spec.tsx (hunk 1 above)
Comment (therealemjy) [reply src/pages/Dashboard/Modals/BorrowRepay/Borrow/index.spec.tsx]: Fixed.

**Pull request merged**